docs/deploy-hyperlane.mdx

```diff
@@ -128,10 +128,11 @@ To send a test message, run:
 ```bash
 hyperlane send message --relay
 ```
-
+:::info
 Currently core self-relay only works **from Holesky** **to your custom chain**. When dispatch is successful, you should see that the message was self-relayed!
 
 Support in the CLI for other origin chains is being added; chains that are not supported will output `Error: No fallback hook config found`.
```

Comment:
I saw a different error message, should we remove the specific error message from here?

Reply:
I think this was fixed by @yorhodes. if this is no longer an issue, we can remove that specific error message. I believe relaying from custom chain to existing chain is still not possible

Reply:
what error did you see @Mo-Hussain ?

Reply:
`Error: Only built 1 of 2 required modules` raised here https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/70830ffb3e0739df0c2030f2e2a0fcdc61e55d5d/typescript/sdk/src/ism/metadata/aggregation.ts#L30

Reply:
Also I think we should callout that self relay from custom chain to aw managed chain will not work

Reply:
> self relay from custom chain to aw managed chain will not work

this will work if there is an ISM configured on the aw managed chain